src/SpacetimeDBClient.cs

```diff
@@ -7,6 +7,7 @@
 using System.Reflection;
 using System.Text;
 using System.Threading;
+using System.Threading.Channels;
```

Comment:
I had to add a nuget package in order to get this working, did you not have to do this?

Reply:
Not with `dotnet build`, but it's possible that pulled in some floating dependency from my system?